apps/token/src/contexts/contracts/contracts-provider.tsx

```diff
@@ -1,29 +1,29 @@
-import type { TxData } from '@vegaprotocol/smart-contracts';
 import {
-  VegaClaim,
-  VegaErc20Bridge,
-  VegaStaking,
-  ERC20Token,
-  VegaVesting,
+  Token,
+  TokenVesting,
+  Claim,
+  CollateralBridge,
+  StakingBridge,
 } from '@vegaprotocol/smart-contracts';
 import { Splash } from '@vegaprotocol/ui-toolkit';
 import { useWeb3React } from '@web3-react/core';
-import uniqBy from 'lodash/uniqBy';
 import React from 'react';
-import { useEnvironment } from '@vegaprotocol/react-helpers';
 
 import { SplashLoader } from '../../components/splash-loader';
 import type { ContractsContextShape } from './contracts-context';
 import { ContractsContext } from './contracts-context';
 import { defaultProvider } from '../../lib/web3-connectors';
+import { useEthereumConfig } from '@vegaprotocol/web3';
+import { useEnvironment } from '@vegaprotocol/react-helpers';
 
 /**
  * Provides Vega Ethereum contract instances to its children.
  */
 export const ContractsProvider = ({ children }: { children: JSX.Element }) => {
-  const { ADDRESSES, VEGA_ENV } = useEnvironment();
   const { provider: activeProvider, account } = useWeb3React();
-  const [txs, setTxs] = React.useState<TxData[]>([]);
+  const { config } = useEthereumConfig();
+  const { VEGA_ENV, ADDRESSES } = useEnvironment();
+  // const [txs, setTxs] = React.useState<TxData[]>([]);
```

Comment:
this line looks like it can be removed

Reply:
See comment here https://github.com/vegaprotocol/frontend-monorepo/pull/495#discussion_r889942639